ramls/schemas/jobExecution.json

```
@@ -0,0 +1,61 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "description":"Job Execution Schema",
+  "type":"object",
+  "properties":{
+    "id":{
+      "description":"Unique identifier",
+      "$ref":"../../ramls/raml-util/schemas/uuid.schema"
+    },
+    "hrId":{
+      "description":"Human readable id",
+      "type":"string"
+    },
+    "fileName":{
+      "description":"File name",
+      "type":"string"
+    },
+    "jobProfile":{
+      "description":"Related JobProfile information",
+      "type":"object",
+      "$ref":"jobProfile.json"
+    },
+    "recordsCount":{
+      "description":"Number of exported records",
+      "type":"integer"
+    },
+    "endedRunning":{
+      "description":"Date and time when the job execution completed",
+      "type":"string",
+      "format":"date-time"
+    },
+    "runBy":{
```

I think this information can be gathered from metadata , just include 
```suggestion
    "metadata": {
      "type": "object",
      "$ref": "../../../raml-util/schemas/metadata.schema",
      "readonly": true
    }
```

---

The information about user's first name and last name can not be gathered from metadata on UI, let's have a separate field for this